tools/linux/uio_drivers/snd/main.c

```diff
@@ -0,0 +1,372 @@
+#include "log.h"
+#include "stream.h"
+#include "util/atomic.h"
+#include <uio/sound.h>
+#include <sddf/sound/queue.h>
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#define PAGE_SIZE_4K 0x1000
+#define QUEUE_BYTES 0x200000
```

Comment:
This is dependent on the system setup right? If so can you make a comment here.

Reply:
Done in https://github.com/au-ts/libvmm/commit/a62c2d8acf2a00684a8e4330f94793721a6744a2